.azure-pipelines/client.yml

```diff
@@ -10,7 +10,8 @@ trigger:
   - master
 
 variables:
-  DotNetCoreVersion: '2.1.503'
+  DotNetCoreSDKVersion: '3.0.100-preview5-011568'
```

Comment:
Should we take this opportunity for this template to reference the globals template?

Reply:
I didn't think it was needed given this yml file will go away soon.